02-spatial-data.Rmd

```diff
@@ -671,6 +671,7 @@ crs(new_raster)
 <!--CRSargs(CRS("+init=epsg:4326"))-->
 Note that in contrast to the **sf** package, **raster** only accepts the `proj4string` representation of the coordinate reference system.
 
+<!--
```

Comment:
I present summary functions in chapter 3 (non-spatial attribute data), therefore I have commented this part.

Reply:
Ok